OIPA/iati/models.py

```diff
@@ -597,6 +599,41 @@ class DocumentLinkTitle(models.Model):
         content_type_field='related_content_type',
         object_id_field='related_object_id')
 
+class DocumentSearch(models.Model):
+    document = models.OneToOneField('Document')
+    content = VectorField()
+    text = VectorField()
+    last_reindexed = models.DateTimeField()
+
+class Document(models.Model):
+    document_link = models.OneToOneField(DocumentLink)
```

Comment:
I just would like to make it separated to facilitate Data migration.
Also, it is usefull to define Document Manager separatly and make Document searchable.
I'll check if I can make it in DocumentLink.